glide-core/src/client/value_conversion.rs

```diff
@@ -2445,6 +2471,139 @@ mod tests {
         );
     }
 
+    #[test]
+    fn convert_function_stats_standalone_response() {
```

Comment:
would be really nice to see a couple of lines describe what this test is supposed to be doing